containers/python/test_sklearn_cifar_container.py

```diff
@@ -27,6 +30,7 @@ def load_cifar(cifar_location, cifar_filename = "train.data", norm=False):
         Z = Z.T
     return (Z, y)
 
+
```

Comment:
Why does the Python formatter add seemingly unnecessary extra spaces between function definitions?